fenix/app/metrics.yaml

```diff
@@ -1607,6 +1607,107 @@ metrics:
     metadata:
       tags:
         - China
+  bookmarks_add:
+    type: labeled_counter
+    lifetime: application
+    description: |
+      A counter that indicates how many bookmarks a user has added.
+      
+      The label for this counter is `<source>`.
+      
+      `source` will be: `page_action_menu` as that is the only
+      entry point right now to add bookmarks.
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1836167
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-android/pull/2375
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+      - cgordon@mozilla.com
+      - rzhao@mozilla.com
```

Comment:
Please remove rzhao@mozilla.com on the notification emails.  This applies to the other 3 as well.  Thanks